src/Command/ExportCommand.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Frosh\TemplateMail\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Export mail template from database to file system
+ */
+class ExportCommand extends AbstractCommand
+{
+    /**
+     * Database connection
+     *
+     * @var \Doctrine\DBAL\Connection
+     */
+    protected $databaseConnection;
+
+    /**
+     * Root directory
+     *
+     * @var string
+     */
+    protected $directory;
+
+    /**
+     * Filesystem
+     *
+     * @var \League\Flysystem\Filesystem
+     */
+    protected $filesystem;
+
+    /**
+     * Set repository for mail templates.
+     *
+     * @param \Doctrine\DBAL\Connection $databaseConnection
+     */
+    public function setDatabaseConnection(\Doctrine\DBAL\Connection $databaseConnection): void
+    {
+        $this->databaseConnection = $databaseConnection;
+    }
+
+    /**
+     * @inerhitDoc
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('frosh:template-mail:export')
+            ->setDescription('Export mail templates from database to file system')
+            ->addArgument('directory', InputArgument::REQUIRED, 'Target directory')
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Dry run (only show what is being done)');
+    }
+
+    /**
+     * @inerhitDoc
+     */
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        // Print title.
+        $this->io->title(sprintf('%s (%s)', $this->getDescription(), $this->getName()));
+
+        // Get dry-run flag.
+        $isDryRun = (bool)$input->getOption('dry-run');
+
+        // Handle directory.
+        $this->directory = $input->getArgument('directory');
+        if (!$isDryRun && !$this->initializeFilesystem()) {
+            return 1;
+        }
+
+        // Fetch mail templates.
+        $query = <<<EOT
+SELECT mail_template_translation.*, mail_template_type.technical_name, locale.code AS locale
```

Comment:
We should select only the specific columns we need. Selecting a wildcard can be problematic when someone adds a similar column  to the first table